src/storage/local_storage.cpp

```diff
@@ -294,8 +294,11 @@ static void UpdateChunk(Vector &data, Vector &updates, Vector &row_ids, idx_t co
 	case PhysicalType::DOUBLE:
 		TemplatedUpdateLoop<double>(data, updates, row_ids, count, base_index);
 		break;
+	case PhysicalType::VARCHAR:
+		TemplatedUpdateLoop<string_t>(data, updates, row_ids, count, base_index);
+		break;
 	default:
-		throw Exception("Unsupported type for in-place update");
+		throw Exception("Unsupported type for in-place update: " + TypeIdToString(data.GetType().InternalType()));
```

Comment:
This could just be `data.GetType().ToString()` no?

Reply:
I wasn't quite sure what the difference between `GetType()` and `GetType().InternalType()` is. Since the switch is based on the internal type I thought this might be more useful debugging information, as I initially expected the internal type of my `TEXT` column to `STRING`, but it was actually `VARCHAR`. 

Of course happy to change if I've misunderstood or if `GetType().ToString()` is more useful in this context!